src/commonMain/kotlin/fr/acinq/lightning/payment/IncomingPaymentHandler.kt

```diff
@@ -84,8 +87,10 @@ class IncomingPaymentHandler(val nodeParams: NodeParams, val walletParams: Walle
             nodeParams.nodePrivateKey,
             description,
             PaymentRequest.DEFAULT_MIN_FINAL_EXPIRY_DELTA,
-            invoiceFeatures,
+            nodeParams.features,
```

Comment:
why not `nodeParams.features.initFeatures()` ?

Reply:
Done in https://github.com/ACINQ/lightning-kmp/pull/313/commits/abd2f4c6e67fa4967a1a67a21eca93a61c3ca39a, the feature bits were already filtered inside `PaymentRequest.create` but it still makes sense to do it here as well.